content/microservices/basics.md

```diff
@@ -371,3 +371,36 @@ this.client
 > info **Hint** The `timeout` operator is imported from the `rxjs/operators` package.
 
 After 5 seconds, if the microservice isn't responding, it will throw an error.
+
+#### Custom Message Serialization
```

Comment:
I'm guessing some people may think this chapter will cover how to implement a custom message serialization on the server level too (which is incorrect). I'm not sure what would be a proper way to eliminate this potential confusion with this heading though 😞

Reply:
```suggestion
#### Custom Client Message Serialization
```
Feels a bit heavy, but it might drive the point home